docs/getting-started/bot.md

```diff
@@ -0,0 +1,412 @@
+Atomist can now be put to work, creating projects, editing code,
```

Comment:
Thinking of it, they feel almost like that page could be split into "one page per top-level section" and moved to User-Guides.